src/node_file.cc

```diff
@@ -1246,6 +1246,9 @@ int MKDirpSync(uv_loop_t* loop,
         case UV_EPERM: {
           return err;
         }
+        case UV_EACCES: {
+          return err;
+        }
```

Comment:
Can we group these two together, i.e.

```c++
case UV_EPERM:
case UV_EACCES: {
  ...
}
```

Also, if the `default:` block starts with an `uv_fs_req_cleanup()` call, then this should have one too, right? (ditto for the async version)